### PR TITLE
Minor Updates to the en.json

### PR DESCRIPTION
**Pull request by mcsaeid**

### CODE REVIEW

**Comment by limpbrains** [comment]
It could be both XPUB or public key (address)

**Comment by mcsaeid** [comment]
I see. Well, the text can reflect that, clarifying that you can both enter a public key _or_ an extended public key, and both will work. If you think that is a good idea, I can re-edit and add more clarification. Personally, I have used XPUBs way more than single public keys.

File: loc/en.json [hunk 1]
```diff
@@ -411,7 +411,7 @@
     "export_title": "Wallet Export",
     "import_do_import": "Import",
     "import_error": "Failed to import. Please make sure that the provided data is valid.",
-    "import_explanation": "Write here your mnemonic, private key, WIF, or anything you’ve got. BlueWallet will do its best to guess the correct format and import your wallet. If a public key is entered, we will add it as a Watch-only wallet.",
+    "import_explanation": "Write here your mnemonic, private key, WIF, or anything you’ve got. BlueWallet will do its best to guess the correct format and import your wallet. If an extended public key (XPUB) is entered, we will add it as a watch-only wallet.",
```
Comment (Overtorment) [inline loc/en.json:414]: technically not correct, regular bitcoin addresses also supported

**Review by Overtorment — CHANGES_REQUESTED** [review]

File: loc/en.json (hunk 1 above)
Comment (mcsaeid) [reply loc/en.json]: Correct. I reverted it to the original but added XPUB in there as well. I believe it adds clarification. Regular addresses are not the only way to create watch-only wallets. Let me know what you think.

**Review by Overtorment — APPROVED** [review]

**Review by marcosrdz — APPROVED** [review]

**Review by ncoelho — APPROVED** [review]

File: loc/en.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (mcsaeid) [reply loc/en.json]: I actually like this, and I think it makes more sense. Check this out:

“Write here your mnemonic **phrase**, **public or** private key, WIF, or anything you’ve got. BlueWallet will do its best to guess the correct format and import your wallet. If **an extended public key (XPUB)** is entered, we will add it as a watch-only wallet.”

[the remark this answers is not in the document]
Comment (ncoelho) [reply loc/en.json]: I found it very wordy, and can be stressful. More important than being technically correct, is being concise. Imagine you are user that wants to recover his wallet from seed (that is the main function of this feature), does not understand bitcoin pretty well, and suddenly we are talking about "XPUB". 

We don't need to explain everything it supports and be support precise about extended public keys, or like "electrum format", "bread format", "Aezeed", "lndhub uri" ... The idea you want to pass to the receptor is "ok, got it, it is here I need to write what I have". Let's keep it simple, that's would be my suggestion.

Like:
_Write your seed words, public key, WIF, or anything you’ve got. BlueWallet will do its best to guess the correct format and import your wallet._

If you can make it shorter, better.

[the remark this answers is not in the document]
Comment (mcsaeid) [reply loc/en.json]: My initial thought was that you wanted it to be technically correct, hence including the public and private keys. Of course, shorter is better, although not too short. With the number of new people coming in every day, user confusion is the last one would want.

**Comment by Overtorment** [comment]
@ncoelho merge?

**Comment by ncoelho** [comment]
Merge

image

**Review by ncoelho — APPROVED** [review]

**Pull request merged**